### PR TITLE
Issue/805

### LINKED ISSUE #805: Unassociated Runner not displayed when there are no systems
Need to tweak the display logic

### PR DESCRIPTION
**Pull request by hazmat345**
Fixes #805

### CODE REVIEW

File: src/ui/src/js/controllers/admin_system.js [hunk 1]
```diff
@@ -272,6 +273,11 @@ export default function adminSystemController(
     $scope.runnerResponse = response;
     $scope.runners = response.data;
 
+    // This is kind of messy, but oh well
+    if (responseState($scope.response) === 'empty') {
+      $scope.response = response;
+    }
+
```
Comment (TheBurchLog) [inline src/ui/src/js/controllers/admin_system.js:280]: So what is actually going on here? If nothing is returned, then set the `$scope.response` to an empty list?

**Review by TheBurchLog — COMMENTED** [review]
If my comment is accurate, then I a guess it is approved :man_shrugging:

File: src/ui/src/js/controllers/admin_system.js (hunk 1 above)
Comment (hazmat345) [reply src/ui/src/js/controllers/admin_system.js:280]: Yeah, I'm not super happy about this, but it's at least an improvement until we get the new system admin page.

So you know the big yellow display that says "Whoops, couldn't find that" when you go to the system admin page when there aren't any systems? That's based off of the return code of `$scope.response`. So even if there are unassociated runners we were still showing that big banner.

This change just hides the banner for that specific case.

The root cause of this is that we're combining two "queries" (systems and runners) into one logical grouping. Our frontend isn't that flexible so I did this 😄

**Comment by hazmat345** [comment]
I'm going to merge this - it's not exactly clean, but it does improve the situation until the admin page redesign is done.

**Pull request merged**